### PR TITLE
Add check_migrations management command

### PR DESCRIPTION
**Pull request by Ian-Foote**
* Raise `CommandError` if there are model changes not present in the
  migrations.

### CODE REVIEW

**Comment by Ian-Foote** [comment]
@meshy Something like this could be what we want. There may be some rough edges and things worth tweaking, but I think this is a good start.

**Comment by meshy** [comment]
Looks good

**Comment by Ian-Foote** [comment]
Unfortunately I need to work out how to deal with django 1.6 travis runs.

**Comment by Ian-Foote** [comment]
@KevinEtienne

File: tests/test_migration_check.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from incuna_test_utils.compat import DJANGO_LT_17
+
+if not DJANGO_LT_17:
```
Comment (Ian-Foote) [inline tests/test_migration_check.py]: This is really ugly, but it's simple and works...

**Comment by Ian-Foote** [comment]
@incuna/backend Any more comments/merge?

File: tests/test_migration_check.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    from django.core.management import call_command, CommandError
+    from django.test import modify_settings, TestCase
```
Comment (meshy) [inline tests/test_migration_check.py]: Do these imports need to be within here?

Comment (meshy) [inline tests/test_migration_check.py]: Can we use a skipif instead?

Comment (Ian-Foote) [inline tests/test_migration_check.py]: `modify_settings` is new in `1.7`, so it needs to be here and we can't use a `skipIf` instead. I put everything in the `if` for simplicity.

Comment (meshy) [inline tests/test_migration_check.py]: :+1:

**Pull request merged**